### PR TITLE
Add wherabouts rock

### PR DESCRIPTION
**Pull request by berkayoz**
Add wherabouts rock by @claudiubelu 

Merging #4 and #5

### CODE REVIEW

**Review by bschimke95 — APPROVED** [review]
LGTM

**Review by bschimke95 — APPROVED** [review]
LGTM

**Comment by claudiubelu** [comment]
I see you've moved ``tox.ini`` and it's related files. That means that it will have to be fixed, in particular ``format`` and ``lint`` targets, as they're referring to ``whereabouts-rock/tests/tests`` as it is now, which is incorrect.

However, changing from ``{tox_root}/tests`` to ``{tox_root}`` has other issues as well, it will actually start treating the files from ``.tox/`` virtual environments as well, which is undesirable (and it will fail, because ``codespell`` will fail against its own unit tests), specific folders would have to be targeted, instead of the entire ``{tox_root}``. Or, we could just have ``tox.ini`` in the root directory.

File: tests/tox.ini [hunk 1]
```diff
@@ -15,19 +15,21 @@ pass_env =
...
-    flake8 {tox_root}/tests
-    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/tests --dry
-    isort {tox_root}/tests --profile=black --check
-    black {tox_root}/tests --check --diff
+    codespell {tox_root}/sanity {tox_root}/integration
+    flake8 {tox_root}/sanity {tox_root}/integration
+    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/sanity --dry
+        licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/integration --dry
+    isort {tox_root}/sanity {tox_root}/integration --profile=black --check
+    black {tox_root}/sanity {tox_root}/integration --check --diff
```
Comment (claudiubelu) [inline tests/tox.ini:32]: we should run codespell, flake8, licenceheaders, isort, and black on both ``integration`` and ``sanity`` folders.

Comment (berkayoz) [reply tests/tox.ini:32]: I believe this should be already resolved, have I missed something?

Comment (claudiubelu) [reply tests/tox.ini:32]: From what I see here, only ``licenseheaders`` is ran on both ``integration`` and ``sanity`` folders. The others are only being run on ``integration``. Same on the ``format`` target.

Comment (claudiubelu) [reply tests/tox.ini:32]: Nevermind, I'm blind, both ``sanity`` and ``integration`` are there. :) LGTM.

**Pull request merged**